src/routes/api/articles.js

```diff
@@ -34,18 +43,27 @@ articles.get(
 articles.get(
```

Comment:
It would be good if the route to fetch drafts articles of a logged in user be the following:`/articles/drafts` instead of `/profile/articles/drafts`

Reply:
@rwajon check, it's updated